### ISSUE TITLE
Add company providing mother's room

### ISSUE THREAD

**Issue by EstiShay** [pycascades/www.pycascades.com#130]
CloudBolt Software's sponsorship contribution is the mother's room, so they should be explicitly named on the Accessibility page, to also help people find it.
